tests/rptest/tests/cloud_storage_scrubber_test.py

```diff
@@ -45,6 +47,41 @@
 ]
 
 
+@dataclass(frozen=True)
+class SegmentMeta:
+    base_offset: int
+    committed_offset: int
+    delta_offset: int
+    delta_offset_end: int
+    base_timestamp: int
+    max_timestamp: int
+    size_bytes: int
+    is_compacted: bool
+    archiver_term: int
+    segment_term: int
+    ntp_revision: int
+
+    @staticmethod
+    def from_dict(json: dict):
+        return SegmentMeta(**json)
+
+
+@dataclass(frozen=True)
+class SegmentMetaAnomaly:
+    anomaly_type: str
+    explanation: str
+    at_segment: SegmentMeta
+    previous_segment: SegmentMeta
+
+    @staticmethod
+    def from_dict(json: dict):
```

Comment:
nit: this name shadows the `json` module, might lead to unexpected behavior

Reply:
fixed